File: Python Exams/Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/05. PC Game Shop/05. PCGameShop.py
ceil_games = int(input())
game_list = [ input() for _ in range(ceil_games)]

hearthstone = game_list.count('Hearthstone')
fornite = game_list.count('Fornite')
overwatch = game_list.count('Overwatch')

first_game = hearthstone / ceil_games * 100
second_game = fornite / ceil_games * 100
three_game = overwatch / ceil_games * 100
all_games_sum = abs((first_game + second_game + three_game) - 100)

print(f'Hearthstone - {first_game:.2f}%\nFornite - {second_game:.2f}%\nOverwatch - {three_game:.2f}%')
print(f'Others - {all_games_sum:.2f}%')



'''
4
Hearthstone
Fornite
Overwatch
Counter-Strike
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/05. PC Game Shop/05. PCGameShop.js
function pcGameShop(arg) {
  let gameCeils = +arg[0]

  let hearthstone = arg.filter(el => el === 'Hearthstone').length
  let fornite = arg.filter(el => el === 'Fornite').length
  let overwatch = arg.filter(el => el === 'Overwatch').length

  let firstGame = (hearthstone / gameCeils * 100).toFixed(2)
  let secondGame = (fornite / gameCeils * 100).toFixed(2)
  let threeGame = (overwatch / gameCeils * 100).toFixed(2)
  let allGameSum = Math.abs((+firstGame + +secondGame + +threeGame) - 100).toFixed(2)

  console.log(`Hearthstone - ${firstGame}%\nFornite - ${secondGame}%\nOverwatch - ${threeGame}%`);
  console.log(`Others - ${allGameSum}%`);
}

//pcGameShop(['4', 'Hearthstone', 'Fornite', 'Overwatch', 'Counter-Strike', 'oo'])  
//pcGameShop(['3', 'Hearthstone', 'Diablo 2', 'Star Craft 2'])